packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -390,68 +390,29 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 		return this.runningSummarizer.enqueueSummarize(...args);
 	};
 
-	private async handleSummaryAcks() {
-		let refSequenceNumber = this.runtime.deltaManager.initialSequenceNumber;
-		let ack: IAckedSummary | undefined;
+	/**
+	 * Responsible for receiving and processing all the summaryAcks.
+	 * @param lastAckRefNumber - Before initializing the summarization, if there is a last Ack to be processed, use it.
+	 */
+	private async handleSummaryAcks(lastAckRefNumber: number) {
```

Comment:
Can we move this function into the RunningSummarizer so that all the handleSummaryAck logic is in one place. It's very hard to follow whats going on since its distributed in 2 different places.
All that is needed it to pass in `this.runtime.deltaManager.initialSequenceNumber` to running summarizer.

Reply:
not even that as we pass the runtime.